include/llama.h

```diff
@@ -510,7 +510,7 @@ extern "C" {
     LLAMA_API uint64_t llama_model_size(const struct llama_model * model);
 
     // Get the default chat template. Returns nullptr if not available
-    LLAMA_API const char * llama_model_chat_template(const struct llama_model * model);
+    LLAMA_API const char * llama_model_chat_template(const struct llama_model * model, const char * name);
```

Comment:
Would need to document this change in https://github.com/ggerganov/llama.cpp/issues/9289

Reply:
Added to upcoming changes, thanks